types/dxf/Utils.d.ts

```diff
@@ -6,9 +6,12 @@ export interface ParsedContent {
 
 export type UtilVertex = [number, number];
 
+// [ref](https://github.com/bjnortier/vecks/blob/master/src/Box2.js)
 export interface Box {
     min: Common.Point2D;
     max: Common.Point2D;
     valid: boolean;
+    width?: number;
```

Comment:
These don't look optional in the source. Why are they marked `?` ?

Reply:
Thank you for pointing that out. I have removed the optional types as per your suggestion.